samples/powerstore_v270_k8s_125.yaml

```diff
@@ -0,0 +1,127 @@
+apiVersion: storage.dell.com/v1
```

Comment:
Are all samples of 2.4 removed? i dont see that here.. I see only the Openshift samples replaced
Sample for K8s 124 is missing?

Reply:
I have removed samples for 2.4